extended/src/main/java/io/kubernetes/client/extended/kubectl/KubectlPatch.java

```diff
@@ -44,10 +44,22 @@ public ApiType execute() throws KubectlException {
     refreshDiscovery();
 
     GenericKubernetesApi genericKubernetesApi = getGenericApi();
-    if (ModelMapper.isNamespaced(apiTypeClass)) {
-      return (ApiType) genericKubernetesApi.patch(namespace, name, patchType, patchContent);
-    } else {
-      return (ApiType) genericKubernetesApi.patch(name, patchType, patchContent);
+    try {
+      if (ModelMapper.isNamespaced(apiTypeClass)) {
+        return (ApiType)
+            genericKubernetesApi
+                .patch(namespace, name, patchType, patchContent)
+                .throwsApiException()
+                .getObject();
+      } else {
+        return (ApiType)
+            genericKubernetesApi
+                .patch(name, patchType, patchContent)
+                .throwsApiException()
+                .getObject();
+      }
+    } catch (Exception e) {
```

Comment:
Please catch a specific exception, not all exceptions.

Reply:
please review again